data/stories.yml

```diff
@@ -0,0 +1,38 @@
+version: "2.0"
```

Comment:
Ditto here, document this file with a detailed header, so a developer understands what this file contributes to the overall solution.